docs/user-manuals/cloneset.md

```diff
@@ -172,6 +172,29 @@ So there will be no limit to CloneSet name.
 
 Don't worry. Even if you enable the `CloneSetShortHash`, CloneSet will still recognize and manage the old Pods with full revision label.
 
+## Scale features
+
+### Scale up in streaming style
+
+**FEATURE STATE:** Kruise v1.0.0
+
+Users can specify `ScaleStrategy. MaxUnavailable` to limit the step size of CloneSet **Scaling Up**, so as to minimize the impact on application services.
```

Comment:
`ScaleStrategy.MaxUnavailable` -- no space in it.

Reply:
done